bitcoinkit/src/main/kotlin/io/horizontalsystems/bitcoinkit/BitcoinKit.kt

```diff
@@ -101,7 +103,8 @@ class BitcoinKit : AbstractKit {
             syncMode: SyncMode = SyncMode.Api(),
             confirmationsThreshold: Int = 6,
             bip: Bip = Bip.BIP44,
-            block: Block?
+            block: Block?,
+            authKey: String
```

Comment:
Can this be optional? The `authKey` is only required for testnet. And we can add a header for the constructor saying that `authkey` is needed for testnet

Reply:
The authKey will soon be needed for both `Mainnet` and `Testnet`, but we can make it optional for now if you want?